lib/transcryptor/instance.rb

```diff
@@ -33,10 +33,14 @@ def initialize(adapter)
   # * <tt>:algorithm</tt> - Encryption algorithm (default: +'aes-256-gcm'+).
   #
 
-  def re_encrypt(table_name, attribute_name, old_opts, new_opts, other_columns = [])
+  def re_encrypt(table_name, attribute_name, old_opts, new_opts, re_encrypt_opts = {})
```

Comment:
Method becomes too big, it needs to be refactored. 
Also, `transcryptor_opts` looks better for me.
What about documentation? :)